rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -698,16 +692,7 @@ private BlockResult executeSequential(
             logger.trace("track commit");
 
             long gasUsed = txExecutor.getGasUsed();
-            long totalGasUsedInBucket;
-
-            try {
-                totalGasUsedInBucket = parallelizeTransactionHandler.getGasUsedIn(bucketId.get());
-            } catch (RuntimeException e) {
-                logger.warn("block: [{}] execution was interrupted", block.getNumber());
-                logger.trace("", e);
-                profiler.stop(metric);
-                return BlockResult.INTERRUPTED_EXECUTION_BLOCK_RESULT;
-            }
+            totalGasUsed = parallelizeTransactionHandler.getGasUsedInSequential();
```

Comment:
Rename to `gasUsedInSequential` and add a new variable `gasUsedInBlock += gasUsed`

Reply:
Remember that a transaction fits the block if there is enough gas in the sequential bucket, the totalGasUsed variable is checked on the init() method